jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -499,8 +499,8 @@ int jbmc_parse_optionst::doit()
       ? std::stoul(cmdline.get_value("java-max-input-array-length"))
       : MAX_NONDET_ARRAY_LENGTH_DEFAULT;
   object_factory_params.max_nondet_string_length =
-    cmdline.isset("string-max-input-length")
-      ? std::stoul(cmdline.get_value("string-max-input-length"))
```

Comment:
Seems like it will kill support for the `--string-max-input-length` option, which is still used in our regression tests and on platform.